kili/cli.py

```diff
@@ -12,8 +14,70 @@ def cli():
     """Kili Command line interface."""
 
 
+@cli.group(context_settings=CONTEXT_SETTINGS)
+def project():
+    """Commands to interact with a Kili project"""
+
+
+@project.command(name='import')
+@click.argument('files', type=str, nargs=-1)
+@click.option('--api-key', type=str, help='Your Api Key', envvar='KILI_API_KEY')
+@click.option('--project-id', type=str, help='Id of the project to import assets in')
+@click.option('--exclude', '-e', type=str, multiple=True,
+              help="Files to exclude from the given files")
+@click.option('--frames', type=bool, help="Only for a frame project, import videos as frames")
+@click.option('--fps', type=int, help="Only for a frame project, import videos with this fps")
+#pylint: disable=too-many-arguments
+def import_assets(api_key, project_id, files, exclude, fps, frames):
```

Comment:
Why don't you use type annotation?

Reply:
I thought that it inherited from the type of the decorators above but indeed it seems that it doesn't

Reply:
@theodu You can maybe add them now to not add to the technical dept

Reply:
@crsegerie I added them, you may be do not look at the last commit